river/Seat.zig

```diff
@@ -412,6 +435,7 @@ pub fn addDevice(self: *Self, device: *wlr.InputDevice) void {
     self.wlr_seat.setCapabilities(.{
         .pointer = true,
         .keyboard = self.keyboards.len > 0,
+        // TODO: switch capability
```

Comment:
there is no switch capability or protocol that sends switch events to clients afaik.